designs/administrator/future-openshift/cluster-setup/index.md

```diff
@@ -0,0 +1,45 @@
+---
+parent: Administrator
+version: 4.X
+---
+
+# Cluster Setup
+
+## First time use
+![](img/setup-hint.png)
+When the user first signs into OpenShift after installation they are presented with a hint block that directs them to complete the cluster setup steps. Clicking on the action button in the hint should open the notification drawer.
+
+
+## Behavior
+Cluster setup items appear in the "recommendations" section of the notification drawer. Once a user has clicked on the action of the recommendation or if the cluster detects that the desired state has been reached, the item disappears from the notification drawer.
+
+## Contents
+![](img/setup-open.png)
+**Note: the exact phrasing and action targets of these items is not final**
+- Configure an identity provider
+  - "You are logged in as a temporary administrative user. Update the cluster OAuth configuration to allow others to log in."
+  - The action button for this item should kick off the "add identity provider" form.
+- Configure alert receivers
+  - "Tell OpenShift where to send important alerts about your cluster."
+  - This action should link to alertmanager configuration page
+- Default Ingress Certificate
+  - Replace the default ingress certificate to enable external clients to securely connect to your apps
+  - [This action should link to docs for ingress certificates](https://docs.openshift.com/container-platform/4.5/security/certificates/replacing-default-ingress-certificate.html)
+- Delete kubeadmin user secret
+  - "Once you've created a cluster admin user, delete this secret for best security"
+  - The action on this item should take the user to the secrets page filtered to show only the "kubeadmin"secret.
+
+### Optional Steps
+![](img/setup-optional.png)
+Some recommended setup steps may not be appropriate for every user. In these cases, the recommendation item should contain a "dismiss" button that would remove it from the notification drawer without taking any action.
+The following are examples of useful optional steps:
+- Configure Registry
+  - "Do this if you want to provide your own container image registry for OpenShift to pull from."
+  - [This should link to the docs for registry configuration](https://docs.openshift.com/container-platform/4.5/registry/configuring-registry-operator.html)
+- Install logging operator
+  - "The OpenShift Logging operator provides a sensible solution for storing and accessing logs"
```

Comment:
Does this install? Or is it more around configuring the logging?

Reply:
Currently, OpenShift Logging is an operator that you have to install in order to have full functionality. The logs that OpenShift collects automatically only stick around for a few hours otherwise